Lib/test/test_traceback.py

```diff
@@ -387,17 +387,16 @@ def get_exception(self, callable):
 
     def test_basic_caret(self):
         def f():
-            raise ValueError("basic caret tests")
+            if True: raise ValueError("basic caret tests")
```

Comment:
Haha, this is a bit confusing the first time I just saw it. I understand that this is so the raise doesn't get elided because is fully covered, but maybe we can do the same with a ternary operation `raise ValueError("blech") or ValueError("never executed")`. I don't have a string opinion to be honest, just that is a bit confusing :)